sdk/core/azure-core/test/ut/http.cpp

```diff
@@ -105,12 +105,8 @@ namespace Azure { namespace Core { namespace Test {
         response.GetHeaders(),
         expected2);
 
-    // Response SetHeader overload method to add from string
-    EXPECT_THROW(response.SetHeader("inv(): header"), std::invalid_argument);
```

Comment:
Keep this test with the existing `SetHeader` overload, passing in `inv()` for the key.

Reply:
that overload was removed

Reply:
I understand. We still have the overload that takes two string `key, name` parameters, correct? Do we have a test already where we pass in `inv()` for the key?

`response.SetHeader("inv()", "header");`